### PR TITLE
docs: add retro for 9.0.0-beta.1

### PR DESCRIPTION
**Pull request by jkleinsc**
This is a retro for the delayed release of 9.0.0-beta.1

### CODE REVIEW

File: wg-releases/retros/9.0.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+In the future, there are several key actions we need to take to prevent this from occurring again.
+
+Before cutting the release branch, we will verify that we have successful release builds off of master.
+
+Once the release branch has been cut, we will trigger test release builds against that branch at least 24 hours ahead of the beta release date.
+
+PRs fixing release issues will be tested against release builds before merging those PRs so that we can verify the validity of the fix.
+
+In order to better support debugging and troubleshooting of release builds in the future, we have already updated our CircleCI release process to run more quickly and to be more ammenable to debug release issues.
```
Comment (codebytere) [inline wg-releases/retros/9.0.0-beta.1.md]: nit: `s/ammenable/amenable`

For thoroughness' sake, can we perhaps link the referenced pull requests?

File: wg-releases/retros/9.0.0-beta.1.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+In order to prevent this from happening we could have run test release builds on the 9-x-y branch before February 5th, 2020 to ensure that the release was ready to go.  Additionally, the nightly release off of master before the 9-x-y branch was cut could have been checked to verify that it was successful.  There is also the `#bot-nightly-ci` Slack channel that could have been checked before 9-x-y was cut to verify whether the nightly release builds were building and running properly.
+
+Additionally, we could have saved some time by testing the initial fix against release builds before merging the PR.
+
+### Future Actions
+
+In the future, there are several key actions we need to take to prevent this from occurring again.
+
+Before cutting the release branch, we will verify that we have successful release builds off of master.
```
Comment (codebytere) [inline wg-releases/retros/9.0.0-beta.1.md]: For the onlooker, 'verify' might not be clear; can we instead say something more explicit like:

> "Before cutting the release branch for the new beta release line from `master`, we will trigger release builds on `master` and ensure that that CI passes and that they can be run locally.

File: wg-releases/retros/9.0.0-beta.1.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Delayed Ship Date Retro - Electron `v9.0.0-beta.1`
+## February 5th, 2020
+
+### What Happened
+
+Per our pre-published [release schedule](https://github.com/electron/governance/blob/master/wg-releases/major-version-release-schedule.md#900), the Electron team planned to release the first beta version of Electron 9.0.0 on February 5th, 2020.  We triggered the first release attempt at 1:45 PM PST.  This release failed at 2:44 PM PST.  A fix for this failure was posted at 8:51 AM PST on February 6th, 2020.  
+
+Unfortunately this fix was not complete and a secondary fix was issued at 12:19 PM PST on February 6th, 2020.  In testing the secondary fix, several other issues came to light.  These issues were triaged and successfully resolved by 7:08 AM PST on February 7th.  
+
+Due to it being the weekend, a release wasn't triggered until Monday, February 10th.  9.0.0-beta.1 was successfully released on February 10th, 2020 at 11:05 AM PST.
```
Comment (sofianguy) [inline wg-releases/retros/9.0.0-beta.1.md]:
```suggestion
Due to it being the weekend, the next release wasn't triggered until Monday, February 10th.  9.0.0-beta.1 was successfully released on February 10th, 2020 at 11:05 AM PST.
```

File: wg-releases/retros/9.0.0-beta.1.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+### Incident Causes
+
+The first failure was caused by a change in the location of the chromedriver release asset.  This issue was fixed via a PR but that PR was not tested against a release build before merging.  After merging a test release build was run which revealed that the first fix wasn't sufficient.  A secondary PR was issued to resolve this issue.  
+
+When the secondary fix was tested against a release build, two new issues surfaced:
```
Comment (sofianguy) [inline wg-releases/retros/9.0.0-beta.1.md]: Add an extra line after this. The numbered bullets don't show up correctly without it (the 1. shows up on the same line)

File: wg-releases/retros/9.0.0-beta.1.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+In order to prevent this from happening we could have run test release builds on the `9-x-y` branch before February 5th, 2020 to ensure that the release was ready to go.  Additionally, the nightly release off of `master` before the `9-x-y` branch was cut could have been checked to verify that it was successful.  There is also the `#bot-nightly-ci` Slack channel that could have been checked before `9-x-y` was cut to verify whether the nightly release builds were building and running properly.
+
+Additionally, we could have saved some time by testing the initial fix against release builds before merging the PR.
+
+### Future Actions
+
+In the future, there are several key actions we need to take to prevent this from occurring again.
+
+Before cutting the release branch for the new beta release line from `master`, we will trigger release builds on `master` and ensure that that CI passes and that the releases can be run locally.
```
Comment (MarshallOfSound) [inline wg-releases/retros/9.0.0-beta.1.md]: I'm not sure this is required, we run release builds on `master` every night so in theory we should have known these release builds were dead. We just need to be more aware of the state of the nightlies

File: wg-releases/retros/9.0.0-beta.1.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+Once the release branch has been cut, we will trigger test release builds against that branch at least 24 hours ahead of the beta release date.
+
+PRs fixing release issues will be tested against release builds before merging those PRs so that we can verify the validity of the fix.
```
Comment (MarshallOfSound) [inline wg-releases/retros/9.0.0-beta.1.md]: Proposal: Make release CI builds auto-run on branches that begin with `release-test/*`

File: wg-releases/retros/9.0.0-beta.1.md (hunk 5 above)
Comment (jkleinsc) [reply wg-releases/retros/9.0.0-beta.1.md]: I updated the text to reflect that @MarshallOfSound.  Let me know what you think

**Pull request merged**